[CI] TSDBDocValuesFormatSingleNodeTests testTSDBWithMultiplePrefixByte failing

**Build Scans:**
- [elasticsearch-periodic #12069 / release-tests](https://gradle-enterprise.elastic.co/s/x2bq2jsm3vsgy)
- [elasticsearch-periodic #12069 / release-tests](https://gradle-enterprise.elastic.co/s/df7haiuiyviny)
- [elasticsearch-pull-request #134585 / checkpart1_release-tests](https://gradle-enterprise.elastic.co/s/j474j3cjlctkk)
- [elasticsearch-pull-request #134498 / checkpart1_release-tests](https://gradle-enterprise.elastic.co/s/7lc4juin7znd4)
- [elasticsearch-pull-request #134411 / checkpart1_release-tests](https://gradle-enterprise.elastic.co/s/bwr7ipu253gpm)
- [elasticsearch-pull-request #134376 / checkpart1_release-tests](https://gradle-enterprise.elastic.co/s/df67fjznzvowm)
- [elasticsearch-pull-request #134317 / checkpart1_release-tests](https://gradle-enterprise.elastic.co/s/sjwd4uslbh4ke)

**Reproduction Line:**
```
./gradlew ":server:test" --tests "org.elasticsearch.index.codec.tsdb.es819.TSDBDocValuesFormatSingleNodeTests.testTSDBWithMultiplePrefixByte" -Dtests.seed=EA00124892D927D2 -Dbuild.snapshot=false -Dtests.jvm.argline="-Dbuild.snapshot=false" -Dlicense.key=x-pack/license-tools/src/test/resources/public.key -Dtests.locale=ca-FR -Dtests.timezone=Asia/Makassar -Druntime.java=25
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.index.codec.tsdb.es819.TSDBDocValuesFormatSingleNodeTests),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('testTSDBWithMultiplePrefixByte'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: IndexWriter codec for field [_ts_routing_hash] should return the expected TSDB doc values format instance
Expected: sameInstance(<DocValuesFormat(name=ES8193TSDB)>)
     but: was <DocValuesFormat(name=ES8193TSDB)>
```

**Issue Reasons:**
- [main] 6 consecutive failures in test testTSDBWithMultiplePrefixByte
- [main] 2 consecutive failures in step release-tests
- [main] 5 consecutive failures in step checkpart1_release-tests
- [main] 1 consecutive failures in pipeline elasticsearch-periodic
- [main] 5 consecutive failures in pipeline elasticsearch-pull-request
- [main] 7 failures in test testTSDBWithMultiplePrefixByte (58.3% fail rate in 12 executions)
- [main] 2 failures in step release-tests (50.0% fail rate in 4 executions)
- [main] 5 failures in step checkpart1_release-tests (62.5% fail rate in 8 executions)
- [main] 5 failures in pipeline elasticsearch-pull-request (62.5% fail rate in 8 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -354,9 +354,6 @@ tests:
 - class: org.elasticsearch.xpack.esql.CsvTests
   method: test {csv-spec:floats.values}
   issue: https://github.com/elastic/elasticsearch/issues/145524
-- class: org.elasticsearch.index.codec.tsdb.es819.ES819TSDBDocValuesFormatTests
-  method: testContainsIterator
-  issue: https://github.com/elastic/elasticsearch/issues/145532
 - class: org.elasticsearch.xpack.esql.CsvTests
   method: test {csv-spec:ints.convertStringToBaseIndexGroup6}
   issue: https://github.com/elastic/elasticsearch/issues/145540
@@ -407,18 +404,12 @@ tests:
 - class: org.elasticsearch.xpack.esql.heap_attack.HeapAttackLimitByIT
   method: testTopNByKeyEncoderTooMuchMemory
   issue: https://github.com/elastic/elasticsearch/issues/145627
-- class: org.elasticsearch.index.codec.tsdb.es819.ES819TSDBDocValuesFormatTests
-  method: testBlockWiseBinarySmallValues
-  issue: https://github.com/elastic/elasticsearch/issues/145635
 - class: org.elasticsearch.repositories.azure.AzureBlobContainerRetriesTests
   method: testWriteLargeBlob
   issue: https://github.com/elastic/elasticsearch/issues/145654
 - class: org.elasticsearch.xpack.esql.CsvIT
   method: test {csv-spec:spatial_shapes.convertFromStringParseError}
   issue: https://github.com/elastic/elasticsearch/issues/145655
-- class: org.elasticsearch.index.codec.tsdb.es819.TSDBDocValuesFormatSingleNodeTests
-  method: testTSDBWithMultiplePrefixByte
-  issue: https://github.com/elastic/elasticsearch/issues/145668
 - class: org.elasticsearch.xpack.core.common.notifications.AbstractAuditorTests
   method: testRecreateTemplateWhenDeleted
   issue: https://github.com/elastic/elasticsearch/issues/123232

--- a/server/src/test/java/org/elasticsearch/index/codec/tsdb/es819/TSDBDocValuesFormatSingleNodeTests.java
+++ b/server/src/test/java/org/elasticsearch/index/codec/tsdb/es819/TSDBDocValuesFormatSingleNodeTests.java
@@ -112,7 +112,7 @@ public class TSDBDocValuesFormatSingleNodeTests extends ESSingleNodeTestCase {
         Set<String> expectedFields = Set.of("@timestamp", "hostname", "gauge", "_tsid", "_ts_routing_hash");
         assertDocValuesFormat(
             indexName,
-            ES819TSDBDocValuesFormatFactory.ES_819_3_TSDB_DOC_VALUES_FORMAT_LARGE_NUMERIC_BLOCK,
+            ES819TSDBDocValuesFormatFactory.ES_819_3_TSDB_DOC_VALUES_FORMAT_LARGE_NUMERIC_AND_BINARY_BLOCK,
             expectedFields
         );
         var indexService = getInstanceFromNode(IndicesService.class).indexServiceSafe(resolveIndex(indexName));

